packages/suite/src/hooks/wallet/__fixtures__/useRbfForm.ts

```diff
@@ -218,7 +269,7 @@ export const composeAndSign = [
                 },
                 {
                     address_n: [2147483692, 2147483648, 2147483648, 1, 0],
-                    amount: '9239',
+                    amount: '3239',
```

Comment:
Was wondering where did this change come from, I can see it was originally 3239, but recently changed to 9239 together with doubling the `composeTransactionCalls` :sweat_smile: